### PR TITLE
[New Entry] HTML Elements: `<del>`

### PR DESCRIPTION
**Pull request by Buono-Marco**
### Description

Added a new entry about the HTML element '<del>'

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/elements/terms/del/del.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```html
+<del>
+  <!-- deleted text here -->
+</del>
+```
+
+The **`<del>`** element should only contain text content or other elements meant for text content.
```
Comment (Dusch4593) [inline content/html/concepts/elements/terms/del/del.md:26]:
```suggestion
The `<del>` element should only contain text content or other elements meant for text content.
```

**Bolding** the first use of the concept/term is sufficient. 😄

File: content/html/concepts/elements/terms/del/del.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example
+
+```html
+<html>
+  <head> </head>
+  <body>
+    <p>It show <del>deleted</del> text</p>
```
Comment (Dusch4593) [inline content/html/concepts/elements/terms/del/del.md:34]:
```suggestion
    <p>This is <del>deleted</del> text.</p>
```

Comment (Dusch4593) [reply content/html/concepts/elements/terms/del/del.md:34]: I would make this edit and replace with a new screen `.png` image.

File: content/html/concepts/elements/terms/del/del.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+Title: '<del>'
+Description: 'Used to show deleted portion text by a document, usually rendered in strikethrough text.'
```
[the remark this answers is not in the document]
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:3]: done

File: content/html/concepts/elements/terms/del/del.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Subjects:
+- 'Web Development'
+- 'Web Design'
+  Tags:
+- 'Elements'
+- 'Tags'
+- 'Style'
+  CatalogContent:
+- 'learn-html'
+- 'paths/front-end-engineer-career-path'
```
[the remark this answers is not in the document]
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:13]: Done

File: content/html/concepts/elements/terms/del/del.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Browsers render it showing strike-through text. Its behavior equals the `<s>` element but is semantically different, `<s>` is usually used to show no longer pertinent, inaccurate, or wrong text.
+
+## Syntax
+
+```html
```
[the remark this answers is not in the document]
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:20]: done

File: content/html/concepts/elements/terms/del/del.md (hunk 1 above)
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:26]: done

File: content/html/concepts/elements/terms/del/del.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  Tags:
+- 'Elements'
+- 'Tags'
+- 'Style'
+  CatalogContent:
+- 'learn-html'
+- 'paths/front-end-engineer-career-path'
+---
+
+The **`<del>`** element is used to show a deleted portion of text and is commonly intended to show the changes made to a document while keeping a history of the changes made. Browsers render it showing strike-through text. Its behavior equals the `<s>` element but is semantically different, `<s>` is usually used to show no longer pertinent, inaccurate, or wrong text.
```
[the remark this answers is not in the document]
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:16]: done

File: content/html/concepts/elements/terms/del/del.md (hunk 2 above)
Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:34]: done

Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md:34]: Done

File: content/html/concepts/elements/terms/del/del.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+  Tags:
+  - 'Elements'
+  - 'Tags'
+  - 'Style'
+  CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
```
Comment (Dusch4593) [inline content/html/concepts/elements/terms/del/del.md:13]:
```suggestion
Tags:
  - 'Elements'
  - 'Tags'
  - 'Style'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
```

Just the list items for `Subjects`, `Tags`, and `CatalogContent` are indented by 2 spaces. 😄

Comment (Buono-Marco) [reply content/html/concepts/elements/terms/del/del.md]: done

**Comment by Buono-Marco** [comment]
I hope I didn't a mistake doing a 'push force'

**Review by SSwiniarski — APPROVED** [review]
@Buono-Marco Looks good to me, I don't have any additional comments.

**Review by Dusch4593 — COMMENTED** [review]
@Buono-Marco This looks ready to merge! 🚀

I made a small edit in e735be7205516b9d296692d1e2d6a57f95427c9f where I included text above and below the \```html block in the "Example" section.

**Comment by Dusch4593** [comment]
Thank you and congrats on your first contribution to Codecademy Docs, @Buono-Marco! 🚀 🎊 🎉 🥳

The entry should now be live: https://www.codecademy.com/resources/docs/html/elements/del

Currently, your Codecademy profile is anonymously listed on the entry page. To make it visible, you'll need to ensure your GitHub username is added to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:

Screen Shot 2022-11-15 at 12 15 47 PM

Looking forward to your next contribution(s) to Docs! 😄

**Comment by Buono-Marco** [comment]
> Thank you and congrats on your first contribution to Codecademy Docs, @Buono-Marco! 🚀 🎊 🎉 🥳
> 
> The entry should now be live: https://www.codecademy.com/resources/docs/html/elements/del
> 
> Currently, your Codecademy profile is anonymously listed on the entry page. To make it visible, you'll need to ensure your GitHub username is added to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:
> 
> Screen Shot 2022-11-15 at 12 15 47 PM
> 
> Looking forward to your next contribution(s) to Docs! 😄

Hi @Dusch4593,
I followed your suggestion to resolve the anonymous contribution issue, but it still appears unknown.

**Comment by Buono-Marco** [comment]
> @Buono-Marco Hmmm, and did you log out of Codecademy and back in through your GitHub account?
> 
> Screen Shot 2022-11-16 at 9 33 43 AM

Exactly

**Comment by yangc95** [comment]
@Buono-Marco This looks resolved now!

**Pull request merged**